src/games-spa/src/components/Word/styles.ts

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+import Button from '@material-ui/core/Button';
+
+type Card = {
+  cardType: number;
```

Comment:
Would it make sense to change `cardType` to `color`?

Reply:
Yep I can definitely adjust this.